front/app/components/UI/PopContainer/index.tsx

```diff
@@ -66,28 +66,23 @@ const StyledIcon = styled(Icon)`
   fill: #333;
 `;
 
-type Props = {
+interface Props {
```

Comment:
If you type the component as React.FC<Props> the children will be automatically added to the type and will be typed more accurately than what I see here. But maybe there's a reason to leave it as is?

Reply:
No, didn't think of it & didn't know that! Just changed.